app.js

```diff
@@ -14,8 +15,20 @@ const shortUrlRouter = require('./app/routes/urlShortner')
 const organizationRouter = require('./app/routes/organisation')
 const commentRouter = require('./app/routes/comment')
 const projectRouter = require('./app/routes/project')
+const notificationRouter = require('./app/routes/notification')
 
 const app = express()
+const server = require('http').Server(app)
+
+server.listen(8810)
```

Comment:
port should be a env_variable

Reply:
> can you explain this
> ` await new Promise((resolve) => setTimeout(() => resolve(), 500))`

@vaibhavdaren during the shutdown of superset, somewhere the thread is not waiting for an async system operation to finish. Thus, the operation is delegated to the system but the execution is not blocked or does not wait for the async operation to finish. In the meantime, the test case execution finishes, jest is shutting down and complains about an open handle. So this is making the jest to wait for some time to finish the execution and disable the open handle issue, during ```npm run test```

Reply:
done